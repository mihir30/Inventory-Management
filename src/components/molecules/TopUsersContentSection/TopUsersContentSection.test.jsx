import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDom from "react-dom";
import TopUsersContentSection from './TopUsersContentSection';

describe("Top users content section Component", () => {

    it('renders the component properly', () => {
        const trEl = document.createElement("tr");
        ReactDom.render(<BrowserRouter><TopUsersContentSection id="1" person="Stutee" email="stu@gmail.com" skype="stutss" units="2"></TopUsersContentSection></BrowserRouter>, trEl); 
        
    });

    it('should have the right top users content section component snapshot', () => {
        const tree = renderer.create().toJSON();
        expect(tree).toMatchSnapshot();
    })

});