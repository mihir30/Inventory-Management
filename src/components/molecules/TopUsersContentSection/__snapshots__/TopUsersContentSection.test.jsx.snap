// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top users content section Component should have the right top users content section component snapshot 1`] = `null`;
